src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,21 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Machine> {
+
+    public ExcavatorProducer() {
+    }
+
+    @Override
+    public List<Machine> get() {
```

Comment:
The return type of the `get` method should be `List<Excavator>` instead of `List<Machine>` to match the parameterized type of the `ExcavatorProducer` class.